src/components/header/HeaderUser.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import "bootstrap/dist/css/bootstrap.css";
+import { Link } from 'react-router';
+
+export function HeaderUser({invisible}) {
+    return (
+        <div className={invisible && 'invisible' || 'd-flex alert  align-items-center text-light'}>
+            <p className='alert-link '>
+               Welcome TestUser!
+            </p>
+            <Link to='/'><button className='btn btn-danger btn-sm ml-2 mb-3'>Log Out</button></Link>
```

Comment:
Maybe it will be better to based on state? I say that we can hold var isLogined and render page accordingly to this var.